modules/salt/pages/formulas-suma.adoc

```diff
@@ -0,0 +1,56 @@
+[[formulas-suma]]
+= Formulas Provided by {productname}
+
+Some formulas are provided by default with {productname}.
+Other official formulas can be installed as RPM packages.
+This section contains information about installing formulas provided by {productname} using either the {webui} or RPM packages.
+
+For information about writing custom formulas, see xref:salt:formulas-custom.adoc[].
+
+
+
+== Install Formulas from the {webui}
+
+Formulas provided by {productname} can be installed using the {productname} {webui}.
+
+
+.Procedure: Installing Formulas from the {webui}
```

Comment:
I think we must better differentiate where to _install_ the formula (that's always on the server) and what you then do with the formula (select or activate it for a client).

Reply:
That's a good point, let's rearrange this a bit.